src/main/java/com/hms_networks/americas/sc/extensions/historicaldata/HistoricalDataQueueManager.java

```diff
@@ -450,4 +450,28 @@ private static void UpdateTrackingStartTime(long updateTime)
 
     isFile1CurrTimeTrackerFile = !isFile1CurrTimeTrackerFile;
   }
+
+  /**
+   * Advance Historical FIFO tracking time. The intention is for exception handlers to make the
+   * decision to advance time.
+   *
+   * @throws IOException if unable to read or write files
+   * @throws TimeTrackerUnrecoverableException if both time tracking files are corrupted
+   * @throws CorruptedTimeTrackerException one of the tracking files is corrupted
+   */
+  public static void AdvanceTrackingStartTime()
```

Comment:
Should be advanceTrackingStartTime() (camel case)

Reply:
changed.